frontend/src/pages/application_page/index.js

```diff
@@ -0,0 +1,196 @@
+    { id: 6, name: "Competitions" }
+  ]
+
+  // FIXME: get from backend
+  const roleQuestions = {
+    0: [0,1,2],
+    1: [3,4,5],
+    2: [6,7,8],
+    3: [9,10,11],
+    4: [0,1,2],
+    5: [3,4,5],
+    6: [6,7,8]
+  }
+
+  // FIXME: get from backend
+  const questions = {
+    0: "What scares you about getting older?",
+    1: "Which book are you ashamed not to have read?",
+    2: "What is the worst thing anyone’s said to you?",
+    3: "What did you want to be when you were growing up?",
+    4: "What is your most treasured possession?",
+    5: "What or who is the greatest love of your life?",
+    6: "What does love feel like?",
+    7: "When did you last cry, and why?",
+    8: "What do you consider your greatest achievement?",
+    9: "Would you rather have more sex, money or fame?",
+    10: "How would you like to be remembered?",
+    11: "What is the most important lesson life has taught you?"
+  };
+
+  // FIXME: handle single question related to many roles (currently gets
+  //        printed under each rolename and all are overwritten)
+  const handleAnswerInput = (e, qID) => {
+    setAnswers({...answers, ...{[qID]: e.target.value}});
+  };
+
+  const toggleRole = (id) => {
+    if (rolesSelected.includes(id)) {
+      setRolesSelected(rolesSelected.filter((role) => role !== id));
+    } else {
+      setRolesSelected((existing) => [...existing, id]);
+    }
+  };
+
+  const onSubmit = () => {
+    // FIXME: make work with backend
+    //        ALSO need to create post submission page and then useNavigate() to it
+    if (rolesSelected.length) {
+      rolesSelected.forEach(role => {
+        const questionsIDs = roleQuestions[role];
+        const responses = questionsIDs.map(id => answers[id]);
+      })
+    } else {
+      alert("Submission failed, you must select at least one role to apply for!");
+    }
+  };
+
+  return (
+    <Container>
+      <CampaignImageCard sx={{boxShadow: 15}}>
+        <CardMedia
+          component="img"
+          height="100%"
+          image={DummyCampaignHeader}
+          alt={`campaign cover for ${1/*FIXME: this*/}`}
+        />
+      </CampaignImageCard>
+      <CampaignDescription>
+        <Typography>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+        </Typography>
+      </CampaignDescription>
+      <FormContainer style={{paddingLeft: "200px", paddingRight: "200px"}}>
+        <Section>
+          <SectionHeader>Applicant Details</SectionHeader>
+          {
+            loggedIn ?
+            <UserInfo>
+              <UserInfoFields>
```

Comment:
I would use a `table` here instead of `div`

OTHERWISE if you cbbs, you could also have:

```
<div>Name: {userInfo.name}</div>
```

or with even better styling:
```
<div style={{display: 'flex', justifyContent: 'space-between'}}>
  <span>Name:</span>
  <span>{userInfo.name}</span>
</div>
```